Docs, of the implementation as it is right this moment (#51)
nodejs/ecmascript-modules: doc/api/cli.md

### `-m`, `--type=type`

> With `--experimental-modules`, this tells Node.js whether to interpret the

Reviewer: ```suggestion
Used with `--experimental-modules`, this configures Node.js to interpret the
```

initial entry point as CommonJS or as an ES module.

> Valid values are `"commonjs"` and `"module"`, where the default is to infer from

Reviewer: ```suggestion
Valid values are `"commonjs"` and `"module"`. The default is to infer from
```

the file extension and the `"type"` field in the nearest parent `package.json`.

> Works with standard file input as well as `--eval`, `--print`, `STDIN`.

Reviewer: ```suggestion
Works for executing a file as well as `--eval`, `--print`, `STDIN`.
```
